actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+*   Make it easier to opt in to `config.force_ssl` and `config.ssl_options` by
+    making them less dangerous to try and easier to disable.
+
+    SSL redirect:
+      * Move `:host` and `:port` options within `redirect: { … }`. Deprecate.
+      * Introduce `:status` and `:body` to customize the redirect response.
+        The 301 permanent default makes it difficult to test the redirect and
+        back out of it since browsers remember the 301. Test with a 302 or 307
+        instead, then switch to 301 once you're confident that all is well.
+
+    HTTP Strict Transport Security (HSTS):
+      * Security. Include the header on http:// responses also. We immediately
+        redirect http:// requests to https://, but the header needs to be set
+        on the initial response, not just the https:// destination.
+      * Shorter max-age. Shorten the default max-age from 1 year to 18 weeks,
+        the minimum to qualify for inclusion in browser preload lists.
```

Comment:
Since `hsts: false` now sends `expires: 0`, we're OK to safely use longish max-age. I considered using 180 days since that's the low end for https://www.ssllabs.com/ssltest/